fixtures/CAPI.ts

```diff
@@ -399,4 +399,5 @@ export const CAPI: CAPIType = {
     contentType: 'Article',
     hasRelated: false,
     hasStoryPackage: false,
+    beaconURL: '//fake.url',
```

Comment:
Is this just til we pass this data?

Reply:
It's in a test, and I ummed and arred about checking this publicly accessible url into the repo.